src/core/src/CoreMain.py

```diff
@@ -101,6 +101,7 @@ def __init__(self, argv):
                     patch_installer.mark_installation_completed()
                     overall_patch_installation_operation_successful = True
                 self.update_patch_substatus_if_pending(patch_operation_requested, overall_patch_installation_operation_successful, patch_assessment_successful, configure_patching_successful, status_handler, composite_logger)
+            status_handler.log_truncated_packages()
```

Comment:
This is not the last place that writes to status. update_patch_substatus_if_pending() in Exception block also writes to status file, making it a potential point where status truncation may occur.

Move this log under finally block

Reply:
move to inside of the finally block, done